lib/tasks/change_note.rake

```diff
@@ -0,0 +1,37 @@
+namespace :change_note do
+  desc "List change notes for a content ID"
+  task :list, %i[content_id] => :environment do |_, args|
+    change_history = Document
+      .find_by(content_id: args[:content_id])
+      .editions
+      .where(minor_change: false)
+      .pluck(:id, :major_change_published_at, :change_note)
+
+    change_history.each do |change|
+      puts change.join("\t")
+    end
+  end
+
+  desc "Amend a change note for an edition"
+  task :amend, %i[edition_id new_change_note email] => :environment do |_, args|
+    edition = Edition.find(args[:edition_id])
+    user_id = User.find_by(email: args[:email]).id
+    old_change_note = edition.change_note
+
+    # rubocop:disable Rails/SkipsModelValidations
+    edition.update_attribute(:change_note, args[:new_change_note])
+    # rubocop:enable Rails/SkipsModelValidations
+
+    EditorialRemark.create!(
+      edition: edition,
+      body: "Updated change note from #{old_change_note} to #{args[:new_change_note]}",
+      author_id: user_id,
+      created_at: Time.zone.now,
+      updated_at: Time.zone.now,
+    )
+
+    PublishingApiDocumentRepublishingWorker.perform_async_in_queue(edition.document.id)
```

Comment:
Do we not need to specify a `queue` here? I'm not very familiar with publishing pipeline.

Reply:
I don't think we do, unless we are bulk republishing.

Reply:
Actually, I think a queue name is needed with `perform_async_in_queue`.  I'll change this to `perform_async` so it uses the default publishing queue.